### PR TITLE
Feature/resource count interface

### PR DESCRIPTION
**Pull request by andreasfrisch**
closes dfds/cloudplatform/issues/1924 dfds/cloudplatform/issues/1925

# Additional Review Notes
Goes beyond the wording of the issue, but follows our talks; Adds
- Simple columnal information in overview
- Detailed information in details-view; Renders only 'relevant' items, and there are no icons as the AWS-provided ones feels too flimsy for my taste.

- Currently uses a simple resources-as-numbers approach instead of graphs or other visualizations. This was discussed offline, to no final conclusion. For now, let us get it out there, so we can gather some feedback and change where needed.

Screenshot 2023-09-04 083715
Screenshot 2023-09-04 084926
Screenshot 2023-09-04 084934

### CODE REVIEW

**Comment by SEQUOIIA** [comment]
@andreasfrisch Interacts badly with modals how? 👀

File: src/src/AppContext.js [hunk 1]
```diff
@@ -90,17 +91,33 @@ function AppProvider({ children }) {
     });
   }
 
+  function updateResourcesCount() {
+    //FLUTTERSHY; see updateCapabilityCosts
```
Comment (kralle333) [inline src/src/AppContext.js:95]: do we want to remove this comment? :)

File: src/src/AppContext.js [hunk 2]
```diff
@@ -90,17 +91,33 @@ function AppProvider({ children }) {
...
     return () => clearInterval(interval);
   }, []);
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      updateResourcesCount();
+    }, 1000 * 60);
+    return () => clearInterval(interval);
+  }, []);
+
```
Comment (kralle333) [inline src/src/AppContext.js:120]: do we want to merge these statements?

File: src/src/MetricsWrapper.js [hunk 3]
```diff
@@ -132,6 +132,36 @@ export class MetricsWrapper {
     return counts;
   }
 
+  pprintAwsResourceCountsForCapability(capabilityId) {
```
Comment (kralle333) [inline src/src/MetricsWrapper.js:135]: pprint?

File: src/src/MetricsWrapper.js [hunk 4]
```diff
@@ -132,6 +132,36 @@ export class MetricsWrapper {
...
+    const counts = metric.capabilitiesMap.get(capabilityId);
+    if (counts === undefined) {
+      return false;
+    }
+    let total = 0;
+    for (let [key, value] of counts) {
+      total += value;
+    }
+    return true;
+  }
```
Comment (kralle333) [inline src/src/MetricsWrapper.js:147]: not completely sure what this function does. Is it debug code that shouldve been removed?

**Review by kralle333 — CHANGES_REQUESTED** [review]
can you double check the code i commented on?

File: src/src/AppContext.js (hunk 1 above)
Comment (andreasfrisch) [reply src/src/AppContext.js:95]: of course not. Removed.

File: src/src/MetricsWrapper.js (hunk 3 above)
Comment (andreasfrisch) [reply src/src/MetricsWrapper.js:135]: Yeah. This is some debugging I didn't clean up.
I used it to pretty-print resource counts in the console to check various things.

I'll remove it

**Review by kralle333 — APPROVED** [review]
Looks GREAT!

**Pull request merged**